apps/ex_wire/lib/ex_wire/kademlia_config.ex

```diff
@@ -0,0 +1,62 @@
+defmodule ExWire.KademliaConfig do
```

Comment:
What do you think about moving this to `ExWire.Kademlia::Config` so we can push as much under `Kademlia` as possible?